up.sh

```diff
@@ -0,0 +1,7 @@
+#Stop any currently running versions of the app
+sudo docker-compose down
+#Build the container, and if successful, start the container
+sudo docker-compose build && sudo docker-compose up -d
+#Display the list of runnng containers for verification
+sudo docker ps
+
```

Comment:
Could you please add either a toggle flag to this script for not using detached mode, or create an alternate script that builds the container and runs it in `stdout`? Seeing the logs in my terminal is more useful for me while debugging.

Reply:
This script is still in development.

Reply:
@nlMeminger Okay. This also requires running Docker as root though. This script might work in your environment, but it will not in others'. Something more general-purpose would be useful. We can talk about this one during the team meeting today.